src/main/java/com/example/pokebattlez/battle/model/Nature.java

```diff
@@ -3,11 +3,11 @@
 import static com.example.pokebattlez.battle.model.Stat.*;
 
 public enum Nature {
-    Hardy(ATTACK, ATTACK), Lonely(ATTACK, DEFENCE), Adamant(ATTACK, SP_ATTACK), Naughty(ATTACK, SP_DEFENCE), Brave(ATTACK, SPEED),
-    Bold(DEFENCE, ATTACK), Docile(DEFENCE, DEFENCE), Impish(DEFENCE, SP_ATTACK), Lax(DEFENCE, SP_DEFENCE), Relaxed(DEFENCE, SPEED),
-    Modest(SP_ATTACK, ATTACK), Mild(SP_ATTACK, DEFENCE), Bashful(SP_ATTACK, SP_ATTACK), Rash(SP_ATTACK, SP_DEFENCE), Quiet(SP_ATTACK, SPEED),
-    Calm(SP_DEFENCE, ATTACK), Gentle(SP_DEFENCE, DEFENCE), Careful(SP_DEFENCE, SP_ATTACK), Quirky(SP_DEFENCE, SP_DEFENCE), Sassy(SP_DEFENCE, SPEED),
-    Timid(SPEED, ATTACK), Hasty(SPEED, DEFENCE), Jolly(SPEED, SP_ATTACK), Naive(SPEED, SP_DEFENCE), Serious(SPEED, SPEED);
+    HARDY(ATTACK, ATTACK), LONELY(ATTACK, DEFENCE), ADAMANT(ATTACK, SPECIAL_ATTACK), NAUGHTY(ATTACK, SPECIAL_DEFENCE), BRAVE(ATTACK, SPEED),
+    BOLD(DEFENCE, ATTACK), DOCILE(DEFENCE, DEFENCE), IMPISH(DEFENCE, SPECIAL_ATTACK), LAX(DEFENCE, SPECIAL_DEFENCE), RELAXED(DEFENCE, SPEED),
+    MODEST(SPECIAL_ATTACK, ATTACK), MILD(SPECIAL_ATTACK, DEFENCE), BASHFUL(SPECIAL_ATTACK, SPECIAL_ATTACK), RASH(SPECIAL_ATTACK, SPECIAL_DEFENCE), QUIET(SPECIAL_ATTACK, SPEED),
+    CALM(SPECIAL_DEFENCE, ATTACK), GENTLE(SPECIAL_DEFENCE, DEFENCE), CAREFUL(SPECIAL_DEFENCE, SPECIAL_ATTACK), QUIRKY(SPECIAL_DEFENCE, SPECIAL_DEFENCE), SASSY(SPECIAL_DEFENCE, SPEED),
+    TIMID(SPEED, ATTACK), HASTY(SPEED, DEFENCE), JOLLY(SPEED, SPECIAL_ATTACK), NAIVE(SPEED, SPECIAL_DEFENCE), SERIOUS(SPEED, SPEED);
```

Comment:
Great job standardizing enum values.

Reply:
This was the plan from the start, just didn't know IntelliJ had a way to capitalize everything at once :)